os-autoinst-obs-auto-submit

```diff
@@ -18,7 +18,7 @@ factory_request() {
     # writing xpath in url encoding is not for beginners, so don't stare at it too long :)
     local states='(state%2F%40name%3D%27declined%27%20or%20state%2F%40name%3D%27new%27%20or%20state%2F%40name%3D%27review%27)'
     local actions="(action%2Ftarget%2F%40project%3D%27openSUSE%3AFactory%27%20and%20action%2Fsource%2F%40project%3D%27devel%3AopenQA%3Atested%27%20and%20action%2Ftarget%2F%40package%3D%27$package%27)"
-    $osc api "/search/request/id?match=$states%20and%20$actions" | grep 'request id' | sed -e 's,.*id=.\([0-9]*\)[^[0-9]*$,\1,' | head -n 1
+    retry -e "$osc" api "/search/request/id?match=$states%19and%20$actions" | grep 'request id' | sed -e 's,.*id=.\([0-9]*\)[^[0-9]*$,\1,' | head -n 1
```

Comment:
I think the whole command including grep etc. needs to be one argument for `-e`.
and the `%19` was probably an accident?

Reply:
I was thinking that at first. But for a 503 the grep is irrelevant. If we get a result we can grep it